WebApplication1/Controllers/TestController.cs

```diff
@@ -45,34 +44,41 @@ public IEnumerable<string> Get()
         [HttpPost]
         public async Task<IActionResult> Post([FromQuery] string hookUrl)
         {          
-            foreach (var item in GetAddresses())
+            foreach (IPAddress address in GetAddresses())
```

Comment:
At anytime do you like to use var in foreach loops or would you always prefer fully types names?